api/v1beta1/cloudstackmachine_types.go

```diff
@@ -17,19 +17,16 @@ limitations under the License.
 package v1beta1
 
 import (
-	"fmt"
-	"strings"
-
-	csCtrlrUtils "github.com/aws/cluster-api-provider-cloudstack/controllers/utils"
-	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	capiv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
 const (
 	// The presence of a finalizer prevents CAPI from deleting the corresponding CAPI data.
 	MachineFinalizer = "cloudstackmachine.infrastructure.cluster.x-k8s.io"
+	ProAffinity      = "pro"
```

Comment:
These affinity consts feel to me like they should be defined in cloudstackaffinitygroup_types.go or affinity_group.go.

Reply:
I'll move them if you really want me to, but let me make my case briefly.

These are the settings on the machine. The affinity group itself uses a different nomenclature.

An affinity group doesn't even make sense as Type "no".

These really are machine specific.

Reply:
Understood and agreed. Makes sense.